app/models/pom_caseload.rb

```diff
@@ -42,6 +40,6 @@ def load_allocations
         allocation,
         offender
       )
-    }
+    }.select(&:valid?)
```

Comment:
Really unsure about this filtering. The warnings from the presenter will just end up in a log which we never look at. Also 'presenters' are just supposed to present, sorting and filtering is normally done in the controller.